examples/pennylane_run_advanced_usage.py

```diff
@@ -142,3 +142,38 @@ def circuit3(param, fixed=None):
 # --> 136         value = self.kwarg_values[self.name][self.idx] * self.mult
 #     137         return value
 # TypeError: unsupported operand type(s) for *: 'NoneType' and 'int'
+#
+# QNodes from different interfaces on one Device
+# -----------------------------------------------
+#
+# In addition to allowing multiple quantum nodes on one device,
```

Comment:
```suggestion
```

Reply:
I think I agree with @AroosaIjaz here. interfaces themselves are not an advanced feature, and should get their own tutorial (especially if we plan to deprecate autograd at some point).

This page is generally for advanced use cases that are too small for their own page, but still worth discussing. Using multiple interfaces on a single qnode definitely fits the bill here.

Reply:
k, let's focus on separate tutorial for the 3 interfaces. That is higher priority atm